src/Mustard/Extension/Geant4X/Geometry/HelicalBox.c++

```diff
@@ -156,30 +242,28 @@ HelicalBox::HelicalBox(std::string name,
         AddInOutTwistedFacet(i, 3, true);
     }
 
-    // compute end position and normal
-    const auto Helix{
-        [&](const auto& u) -> G4Point3D {
-            const auto u1{u + phi0};
-            return {radius * std::cos(u1),
-                    radius * std::sin(u1),
-                    u1 * tanAR - zOffset};
-        }};
-    const auto EndFaceNormal{
-        [&](const auto& u) -> G4ThreeVector {
-            const auto u1{u + phi0};
-            return {-radius * std::sin(u1),
-                    radius * std::cos(u1),
-                    tanAR};
-        }};
-    fFrontEndPosition = Helix(0);
-    fFrontEndNormal = EndFaceNormal(0).unit();
-    fBackEndPosition = Helix(phiTotal);
-    fBackEndNormal = EndFaceNormal(phiTotal).unit();
-
     // seal front end
-    AddFacet(new G4QuadrangularFacet{x(0, 0), x(0, 1), x(0, 2), x(0, 3), ABSOLUTE});
+    if (frontParallel) {
```

Comment:
Same, if `CreateParallelPoints` can be done in `MainPoint` these may not have to be changed.